Knittery/Network/NetworkHandler+Extension.swift

```diff
@@ -34,6 +34,32 @@ extension NetworkHandler {
         }
     }
     
+    // TODO: Make a data type and helper to construct queries, maybe factory? will cut down on optionals as well
+    // yarns have fewer search options
+//    pc=hat&sort=best
+    static func requestPatternSearch(query: String? = "", page: Int? = 1, resultHandler: @escaping (Result<PatternSearch, ApiError>) -> Void) {
+        // TODO: force unwrap needs guard or other implementation especially as it is truly optional
+//        let request = domain + query!
```

Comment:
I would add a TODO to look into using URLComponents when you're looking at search. 
Some info here: https://cocoacasts.com/working-with-nsurlcomponents-in-swift